src/lib/y2storage/blk_device.rb

```diff
@@ -217,6 +217,23 @@ def udev_full_ids
     storage_forward :create_blk_filesystem, as: "Filesystems::BlkFilesystem", raise_errors: true
     alias_method :create_filesystem, :create_blk_filesystem
 
+    # @!method create_bcache
+    #   Creates backing device build on current block device.
+    #
+    #   If the blk device has children, the children will become children of
```

Comment:
This sentence looks like a wrong copy&paste

Reply:
why you think so? It is also from libstorage, that it moves its children ( probably partitions or fs ) to bcache from blk_device.

Reply:
NP: I would try to document possible exceptions.

Reply:
I think so because it says "will become children of the encryption device" instead of "will become children of the bcache device".

Reply:
@joseivanlopez libstorage-ng does not mention any exception for this call. Real testing will show reality. For create_bcache_cset it is written, so I will add it.